client/src/App.js

```diff
@@ -339,7 +365,9 @@ render() {
                                     openModalHandler={this.openModalHandler}
                                     facilityModal={this.state.facilityModal}
                                     closeModalHandler={this.closeModalHandler}
-                                    facilityDetails={this.state.facilityDetails}/>
+                                    facilityDetails={this.state.facilityDetails}
+                                    onMouseOver={this.onMouseOver}
+                                    onMouseOut={this.onMouseOut}/>
```

Comment:
This is what I was thinking. I just thought we do not need to pass this from App.js, though.

Reply:
No need to change anything. This is just a comment.

Reply:
I could have done this at the RecyclePage level but made more sense to me to keep all functions in App.js. Not sure what is correct process.